packages/react-components/src/components/CameraButton.tsx

```diff
@@ -139,44 +148,62 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
   const [waitForCamera, setWaitForCamera] = useState(false);
   const localeStrings = useLocale().strings.cameraButton;
   const strings = { ...localeStrings, ...props.strings };
-  const onRenderCameraOnIcon = (): JSX.Element => (
-    <HighContrastAwareIcon disabled={props.disabled || waitForCamera} iconName="ControlButtonCameraOn" />
-  );
-  const onRenderCameraOffIcon = (): JSX.Element => (
-    <HighContrastAwareIcon disabled={props.disabled || waitForCamera} iconName="ControlButtonCameraOff" />
-  );
+  const [announcerString, setAnnouncerString] = useState<string>(strings.cameraActionTurnedOffAnnouncement);
+  const onRenderCameraOnIcon = (): JSX.Element => {
+    return <HighContrastAwareIcon disabled={props.disabled || waitForCamera} iconName="ControlButtonCameraOn" />;
+  };
+  const onRenderCameraOffIcon = (): JSX.Element => {
+    return <HighContrastAwareIcon disabled={props.disabled || waitForCamera} iconName="ControlButtonCameraOff" />;
+  };
   if (waitForCamera && strings.tooltipVideoLoadingContent) {
     strings.tooltipDisabledContent = strings.tooltipVideoLoadingContent;
   }
 
   const onToggleClick = useCallback(async () => {
     // Throttle click on camera, need to await onToggleCamera then allow another click
     if (onToggleCamera) {
+      // allows for the setting of narrator strings triggering the announcer when camera is turned on or off.
+      if (announcerString === strings.cameraActionTurnedOffAnnouncement) {
```

Comment:
This toggle will be incorrect if the component starts with the camera already enabled. Instead of checking the string, can we check the state of the button? (maybe props.checked?)

Reply:
Difficulty here is that we want it to almost be the opposite of props.checked...
I'll let you explore what else in the interface could be best for this, but one suggestion is to revert back the idea of having:

```ts
const announcerString = props.checked ? strings.cameraActionTurnedOnAnnouncement : strings.cameraActionTurnedOffAnnouncement
```

Then to fix the issue of it being read on first render have:
```ts
const [buttonClicked, setButtonClicked] = useState(false);

// Only set the announcer string if the button has been clicked at least once
const announcerString = !buttonClicked ? undefined : props.checked ? strings.cameraActionTurnedOnAnnouncement : strings.cameraActionTurnedOffAnnouncement

onToggleClick() => {
  ...
  await onToggleCamera();
  setButtonClicked(true);
}
```

Reply:
Actually that won't work if the button is clicked and is in the waitForCamera state... I'll stop trying to post solutions I haven't tested...